osidb/models.py

```diff
@@ -2110,7 +2245,6 @@ def validate(self, *args, **kwargs):
 
 
 class CVEv5PackageVersions(PackageVersions):
-
```

Comment:
All these black line removals are unrelated to the task so should be either done in an independent commit with an explanation why we remove them or not done at all.